opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -249,7 +262,11 @@ def _init_logging(
 
     handler = LoggingHandler(level=logging.NOTSET, logger_provider=provider)
 
-    logging.getLogger().addHandler(handler)
+    logging.basicConfig(
```

Comment:
How does this fit with what is done in `opentelemetry-instrumentation-logging` with `OTEL_PYTHON_LOG_CORRELATION=true`?

Reply:
I have not tested this yet. I believe basic config can only be set once. Since instrumentations are loaded after configurators, autoinstrumentation would take priority.

Reply:
It seems the logging instrumentation is not supposed to be used with the log handler anyways. So, if someone uses those environment variables in auto instrumentation, I think it would make that this will occur first and take precedent. In auto instrumentation, the sdk is configured before instrumentations.